react/src/pages/EnvironmentPage.tsx

```diff
@@ -1,37 +1,53 @@
 import ContainerRegistryList from '../components/ContainerRegistryList';
 import Flex from '../components/Flex';
-import { useSuspendedBackendaiClient } from '../hooks';
+import { useSuspendedBackendaiClient, useWebUINavigate } from '../hooks';
 import { theme } from 'antd';
 import Card from 'antd/es/card/Card';
-import { useState, Suspense } from 'react';
+import { Suspense } from 'react';
 import { useTranslation } from 'react-i18next';
+import { StringParam, useQueryParam, withDefault } from 'use-query-params';
+
+const tabParam = withDefault(StringParam, 'image');
 
-type TabKey = 'imageList' | 'presetList' | 'registryList';
 const EnvironmentPage = () => {
   const { t } = useTranslation();
-  const [curTabKey, setCurTabKey] = useState<TabKey>('imageList');
+  const webUINavigate = useWebUINavigate();
+  const [curTabKey] = useQueryParam('tab', tabParam);
   const baiClient = useSuspendedBackendaiClient();
   const isSupportContainerRegistryGraphQL = baiClient.supports(
     'container-registry-gql',
   );
   const { token } = theme.useToken();
+
   return (
     <Card
       activeTabKey={curTabKey}
-      onTabChange={(key) => setCurTabKey(key as TabKey)}
+      onTabChange={(key) => {
+        webUINavigate(
+          {
+            pathname: '/environment',
+            search: `?tab=${key}`,
```

Comment:
Please use `new URLSearchParams` instead of string manipulation. Please refer to [this link](https://github.com/lablup/backend.ai-webui/blob/main/react/src/components/FolderExplorerOpener.tsx#L28-L35).